gnome-image-installer/pages/install/gis-scribe.c

```diff
@@ -764,7 +773,10 @@ gis_scribe_begin_verify (GisScribe *self,
       "--input-size-hint", size_str,
       "--verify", signature_path, image_path, NULL
   };
-  gint outfd;
+  g_autoptr(GSubprocessLauncher) launcher = NULL;
+  g_autoptr(GSubprocess) subprocess = NULL;
+  GInputStream *gpg_stdout = NULL;
```

Comment:
I think the presence of `g_autoptr` on the variables which do own a reference is a good suggestion that this one does not. It's a moot point because the variable is not referenced before it is assigned to so the compiler will probably drop the `= NULL`s…

Reply:
> I think the presence of `g_autoptr` on the variables which do own a reference is a good suggestion that this one does not.

Yes, except there are functions where you’ve used `GError *error = NULL` without `g_autoptr()` (unless that’s a convention you changed in the latter half of the branch?).

> It's a moot point because the variable is not referenced before it is assigned to so the compiler will probably drop the `= NULL`s…

(I don’t care what the compiler does! I care about the code readability here.)